addons/website_slides/models/slide_channel.py

```diff
@@ -401,8 +401,10 @@ def toggle_active(self):
         to_archive = self.filtered(lambda channel: channel.active)
         to_activate = self.filtered(lambda channel: not channel.active)
         res = super(Channel, self).toggle_active()
+        super(Channel, self).write({'is_published': False})
         if to_archive:
             to_archive.mapped('slide_ids').action_archive()
+            to_archive.mapped('slide_ids').action_unpublish()
```

Comment:
Well maybe this should be done directly in write: unpublishing a channel will unpublish its slides. Or do you want it specific to archive flow, and it that case why ?

Reply:
The task was only for the archive flow. But maybe it make sens to unpublish the slides when you unpublish en whole Channel.

Reply:
I would move it to the write directly; you don't archive often and if you do it better do it completely I think :) .